### PR TITLE
Add Env Var to disable second level cache

### PR DESCRIPTION
**Pull request by gnattu**
**Changes**

This is an attempt to track down possible causes of remaining database lockups. Add an environment variable to disable the second-level cache entirely to see if it works better on systems that still experience lockups.

For example (note the double underscore `__`  between `SQLITE` and `disableSecondLevelCache` is required):

```
export  JELLYFIN_SQLITE__disableSecondLevelCache=true
```

This is not a guaranteed fix, just an option for (advanced) users to test if it improves the situation. So it may be removed in future version.

**Issues**

See #11589 #11813 #11834

Probably also others I'm not aware of.

### CODE REVIEW

File: MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs [hunk 1]
```diff
@@ -128,5 +133,26 @@ public static bool UseUnixSocket(this IConfiguration configuration)
...
             => configuration.GetValue<int?>(SqliteCacheSizeKey);
+
+        /// <summary>
+        /// Gets whether second level cache disabled from the <see cref="IConfiguration" />.
+        /// </summary>
+        /// <param name="configuration">The configuration to read the setting from.</param>
+        /// <returns>Whether second level cache disabled.</returns>
+        public static bool GetSqliteSecondLevelCacheDisabled(this IConfiguration configuration)
+        {
+            var disableSecondLevelCacheConfig = configuration.GetValue<string?>(SqliteDisableSecondLevelCacheKey);
```
Comment (cvium) [inline MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs:144]: Can't we use `GetValue<bool>(SqliteDisableSecondLevelCacheKey, false)`?

Comment (gnattu) [reply MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs:144]: We can, but that will make this mush less user-friendly, as the user will have to type in a valid bool string (TRUE or FALSE, case insensitive) and all other value will make dotnet complain and error out. This is more user friendly.

Comment (crobibero) [reply MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs:144]: This shouldn't be a user-friendly config option, if they change the config then they must conform to the accepted values.

Comment (gnattu) [reply MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs:144]: But not even supporting 1 and 0 which is super common for env vars is a little bit too restrictive.

**Review by Shadowghost — APPROVED** [review]

File: MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs (hunk 1 above)
Comment (joshuaboniface) [reply MediaBrowser.Controller/Extensions/ConfigurationExtensions.cs]: I'd agree with gnattu here, sure it's not *common* to use but making it deliberately obtuse to use is not good design. If this is needed to get sensible boolean-like flags (e.g. `true`, `TRUE`, `1`, etc.) then I think it's worthwhile.

**Review by 1337joe — APPROVED** [review]

**Review by crobibero — APPROVED** [review]

**Pull request merged**